### PR TITLE
test(regex): add ContentEq tests

### PR DESCRIPTION
**Pull request by DonIsaac**
These files have no test coverage. This should help.

### CODE REVIEW

**Comment by DonIsaac** [comment]
* **#6409** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6409?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @DonIsaac and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/oxc_regular_expression/src/generated/test.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+fn parse<'a>(alloc: &'a Allocator, regex: &'a str) -> Pattern<'a> {
+    Parser::new(alloc, regex, ParserOptions::default()).parse().unwrap()
+}
+
+fn test_cases() -> impl Iterator<Item = (&'static str, &'static str)> {
+    use std::iter;
+    let cases = vec!["abc", "^abc?", "[a-z]", "a{1,2}", "a{1,}", "a(?=b)"];
+
+    // let alloc = Allocator::default();
```
Comment (camchenry) [inline crates/oxc_regular_expression/src/generated/test.rs:15]: Delete this now?

**Review by camchenry — APPROVED** [review]

File: crates/oxc_regular_expression/src/generated/test.rs (hunk 1 above)
Comment (Boshen) [inline crates/oxc_regular_expression/src/generated/test.rs:15]:
```suggestion
```

**Comment by Boshen** [comment]
Nice catch, I found a better place to assert this, and we caught a few bugs already.

**Comment by DonIsaac** [comment]
Closing in favor of #6411

**Pull request closed without merging**